src/script/media/MediaStreamErrorTypes.ts

```diff
@@ -29,3 +29,17 @@ export const MEDIA_STREAM_ERROR_TYPES = {
   ],
   PERMISSION: [MEDIA_STREAM_ERROR.NOT_ALLOWED_ERROR, MEDIA_STREAM_ERROR.SECURITY_ERROR],
 };
+
+export function isMediaStreamDeviceError(errorStr: string): boolean {
+  return MEDIA_STREAM_ERROR_TYPES.DEVICE.filter(errorType => errorType == errorStr).length > 0;
```

Comment:
Result will be the same with this fn from suggestion? If yes please use :)

```suggestion
  return MEDIA_STREAM_ERROR_TYPES.DEVICE.includes(errorStr);
```